services/QuillLMS/spec/support/shared_contexts/stripe_integration/stripe_subscription.rb

```diff
@@ -6,8 +6,8 @@
   include_context 'Stripe Customer'
 
   let(:stripe_subscription_id) { "sub_#{SecureRandom.hex}" }
-  let(:current_period_end) { (Date.today + 365).to_time.to_i }
-  let(:current_period_start) { Date.today.to_time.to_i }
+  let(:current_period_end) { (Date.current + 365).to_time.to_i }
```

Comment:
`1.year.from_now.to_i`  No need to use `to_time` here if we do that, I think.

Reply:
I like that much better